cli/src/bin/phylum.rs

```diff
@@ -212,6 +214,11 @@ async fn handle_commands() -> CommandResult {
             let resp = api.cancel(&request_id).await;
             print_response(&resp, true, None);
         }
+    } else if cfg!(feature = "extensions") {
+        #[cfg(feature = "extensions")]
+        if let Some(matches) = matches.subcommand_matches("extension") {
+            return handle_extensions(matches).await;
+        }
```

Comment:
Extensions don't use the API, so this code should be moved up to be before the creation of the `PhylumApi` object. That will allow unauthenticated users to run `extension` commands.

More importantly, that will allow the `assert_cmd` tests to work correctly even without API authentication.